### PR TITLE
fix sample-controller README

### PR DESCRIPTION
**Pull request by CaoShuFeng**
The guide in doc causes an error
```
$ go run *.go -kubeconfig=$HOME/.kube/config
go run: cannot run *_test.go files (controller_test.go)
```

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/sample-controller/README.md [hunk 1]
```diff
@@ -42,7 +42,8 @@ This is an example of how to build a kube-like controller with a single type.
 
 ```sh
 # assumes you have a working kubeconfig, not required if operating in-cluster
-$ go run *.go -kubeconfig=$HOME/.kube/config
+$ go build -o sample-controller .
```
Comment (nikhita) [inline staging/src/k8s.io/sample-controller/README.md]: @CaoShuFeng :+1: thank you!

**Comment by nikhita** [comment]
/lgtm

/assign @sttts 
for approval

**Comment by sttts** [comment]
/approve

The other repos don't have this?

**Comment by nikhita** [comment]
> The other repos don't have this?

No. But didn't understand which other repos. sample-apiserver has it's own complicated instructions. :)

**Comment by sttts** [comment]
> But didn't understand which other repos

One never knows the amount of cargo culting going on :)

**Pull request merged**